packages/atomizer/src/rules.js

```diff
@@ -1777,6 +1772,24 @@ module.exports = [
             'n': 'normal',
             'm': 'manual'
         }]
+    },
+        /**
+    ==================================================================
+    ISOLATE
+    ==================================================================
+    */
+    {
+        'type': 'pattern',
+        'name': 'Isolation',
+        'matcher': 'I',
```

Comment:
`I` is a pretty generic abbreviation for this property. What do you think about something like `Iso` instead?

Reply:
Sounds good to me :)